agent/acs/session/refresh_credentials_responder.go

```diff
@@ -20,6 +20,7 @@ import (
 	"github.com/pkg/errors"
```

Comment:
Can you add the changes made in agent/acs/session to the PR description as well? Thanks!